static/js/portico/landing-page.js

```diff
@@ -158,11 +208,6 @@ const events = function () {
 };
 
 $(() => {
-    // Initiate the bootstrap carousel logic
-    $(".carousel").carousel({
-        interval: false,
-    });
-
     // Move to the next slide on clicking inside the carousel container
     $(".carousel-inner .item-container").on("click", function (e) {
         const get_tag_name = e.target.tagName.toLowerCase();
```

Comment:
Presumably this carousel logic can be removed too?

Reply:
Yeah.